core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -502,7 +502,10 @@ class WorkspaceService(protected val userInfo: UserInfo,
         for {
           _ <- deletePetsInProject(googleProjectId, userInfoForSam)
           _ <- gcsDAO.deleteGoogleProject(googleProjectId)
-          _ <- samDAO.deleteResource(SamResourceTypeNames.googleProject, googleProjectId.value, userInfoForSam)
+          _ <- samDAO.deleteResource(SamResourceTypeNames.googleProject, googleProjectId.value, userInfoForSam).recover {
+            case regrets: RawlsExceptionWithErrorReport if regrets.errorReport.statusCode == Option(StatusCodes.NotFound) =>
```

Comment:
I'm tempted to do the same thing when we delete `workflow-collection` and `workspace` resources. If Sam returns a 404 when we try to delete it, just treat that as okay.